README.md

```diff
@@ -8,7 +8,7 @@
 
 HEAD of master is not stable, for stable version use [this](https://github.com/nem0/LumixEngine/tree/f77a0b6a899be4736f0ce26133bbe8061a3828f1)
 
-3D Game Engine
+3D Game Engine with Vulkan support.
```

Comment:
we did not use full stop here before and I don't think we should start with it now. It's too dangerous.